src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
```

Comment:
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/81781479/219355751-2ae16b5e-8431-41e8-8d6d-aa0f5b14f03c.png">

you should divide logic into 3 methods
```
   String dataFromFile = readFromFile(fromFileName);
   String report = createReport(dataFromFile);
   writeToFile(report, toFileName);